src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
+public class Tagging {
+
+	public final String tagging;
+	
+	public Tagging(String name, String tag, String action) {
+		String actionSign = "";
+		if (action.equals("Add")) {
```

Comment:
How do you ensure that the input for action is what you are expecting (either 'Add' or 'Delete')?